README.md

```diff
@@ -2,6 +2,32 @@
 
 Ever wanted to generate new Dutch slang? Look no further! This application makes use of Recurrent Neural Networks trained on a comprehensive database of Dutch slang to generate new slang. Some words may immediately bring up a visceral feeling of what they mean, and this application allows you to provide the community with your interpretation of what the newly created slang means.
 
+## Examples
+We have several pre-trained models available to generate novel words from multiple classes.
+
+### Straattaal
+- rampa
+- joeko
+- opjo
+- oggor
```

Comment:
```suggestion
- daggie
```
I really love daggie, and oggor is not my favorite

Reply:
Maybe we should also add our attributed meanings and "challenge people to come up with something better" ?

Reply:
Agree with daggie. I have no creativity to add definitions right now. should we format them like 

- **rampa**: een grote ramp

+ which language?

Reply:
Yes, I like the format. 
I'd say we should give both the Dutch and English translation of the meaning. May be able to work on it tomorrow morning, so I created this issue (https://github.com/Sasafrass/straattaal/issues/59). I pushed it to the new Documentation board, so it's no longer a blocker for this PR, and we can perhaps create a new PR for it.